tdrs-frontend/src/components/EditProfile/EditProfile.test.js

```diff
@@ -450,15 +452,39 @@ describe('EditProfile', () => {
         ],
       },
     })
-    const wrapper = mount(
+
+    render(
```

Comment:
Note that we are rewriting enzyme tests with react-testing-library everywhere possible when we encounter them